pkg/kubelet/cm/pod_container_manager_linux.go

```diff
@@ -72,7 +72,7 @@ func (m *podContainerManagerImpl) Exists(pod *v1.Pod) bool {
 func (m *podContainerManagerImpl) EnsureExists(pod *v1.Pod) error {
 	podContainerName, _ := m.GetPodContainerName(pod)
 	// check if container already exist
-	alreadyExists := m.Exists(pod)
+	alreadyExists := m.cgroupManager.Exists(podContainerName)
```

Comment:
If implementation of ```m.Exists``` changes in future we'd need to change this code as well, correct?

Would it make sense to shift `podContainerName, _ := m.GetPodContainerName(pod)` closer to the place where `podcontainerName` variable is used and leave m.Exists call as it is?

Reply:
Thinks for you review.I try to shift podContainerName, _ := m.GetPodContainerName(pod) closer to the place where podcontainerName variable is used.

Reply:
Please reivew again,Think you!

Reply:
/lgtm
looks better